res/ai/DefaultAIPlayer/TaskSchedule.lua

```diff
@@ -1202,54 +1156,6 @@ function TaskSchedule:AddSpotRequisition(broadcastMaterialGUID, guessedAudience,
 
 	return requisition
 end
-
-
-
-
-function TaskSchedule:FixAdvertisement(day, hour)
-	if (TVT.GetAdContractCount() <= 1) and (TVT.GetProgrammeLicenceCount() <= 0)  then
-		--self:LogDebug("FixAdvertisement: " .. day .."/".. string.format("%02d", hour) .. ":55 - NOT POSSIBLE, not enough adcontracts (>1) or licences.")
-	else
-		--self:LogDebug("FixAdvertisement: " .. day .."/".. string.format("%02d", hour) .. ":55")
-
-		--increase importance of schedule task!
-		self.SituationPriority = 75
-
-		-- assign player (if called from outside, this is not set yet)
-		self.Player = getPlayer()
-		-- should start schedule then
-		self.Player:ForceNextTask()
-	end
-end
-
-
-function TaskSchedule:_FixImminentOutage(day, hour, minute, situationPriority)
-	if (TVT.GetAdContractCount() <= 0) and (TVT.GetProgrammeLicenceCount() <= 0) then
-		--self:LogDebug("FixImminentOutage: " .. day .."/".. string.format("%02d", hour) .. ":" .. minute .. " - NOT POSSIBLE, not enough adcontracts or licences.")
-	else
-		--self:LogDebug("FixImminentOutage: " .. day .."/".. string.format("%02d", hour) .. ":" .. minute)
-
-		--increase importance of schedule task!
-		self.SituationPriority = situationPriority
-
-		-- assign player (if called from outside, this is not set yet)
-		self.Player = getPlayer()
-		-- should start schedule then
-		self.Player:ForceNextTask()
-	end
-end
-
-
-function TaskSchedule:FixImminentAdOutage(day, hour)
```

Comment:
Die Funktion wird in DefaultPlayer.lua erwaehnt -- eventuell Kommentar dort anpassen.